README.md

```diff
@@ -3,26 +3,43 @@
 [![Translation Status](https://hosted.weblate.org/widgets/moonlight/-/moonlight-android/svg-badge.svg)](https://hosted.weblate.org/projects/moonlight/moonlight-android/)
 
-[Moonlight for Android](https://moonlight-stream.org) is an open source client for NVIDIA GameStream and [Sunshine](https://github.com/LizardByte/Sunshine).
+[Moonlight](https://moonlight-stream.org/) for Android is an open source client for
+[Nvidia GameStream](https://www.nvidia.com/en-us/support/gamestream/) and
+[Sunshine](https://github.com/LizardByte/Sunshine).
 
-Moonlight for Android will allow you to stream your full collection of games from your Windows PC to your Android device,
-whether in your own home or over the internet.
+Moonlight for Android will allow you to stream your full collection of games from your computer to
+your Android device, whether in your own home or over the internet.
 
-Moonlight also has a [PC client](https://github.com/moonlight-stream/moonlight-qt) and [iOS/tvOS client](https://github.com/moonlight-stream/moonlight-ios).
+Moonlight also has clients for [PC](https://github.com/moonlight-stream/moonlight-qt),
+[iOS/tvOS](https://github.com/moonlight-stream/moonlight-ios), and
+[more](https://moonlight-stream.org/).
 
-You can follow development on our [Discord server](https://moonlight-stream.org/discord) and help translate Moonlight into your language on [Weblate](https://hosted.weblate.org/projects/moonlight/moonlight-android/).
+You can follow development on our [Discord server](https://moonlight-stream.org/discord)
+and help translate Moonlight into your language on
+[Weblate](https://hosted.weblate.org/projects/moonlight/moonlight-android/).
+
+## Download
 
-## Downloads
 * [Google Play Store](https://play.google.com/store/apps/details?id=com.limelight)
 * [Amazon App Store](https://www.amazon.com/gp/product/B00JK4MFN2)
 * [F-Droid](https://f-droid.org/packages/com.limelight)
 * [APK](https://github.com/moonlight-stream/moonlight-android/releases)
 
-## Building
-* Install Android Studio and the Android NDK
-* Run ‘git submodule update --init --recursive’ from within moonlight-android/
-* In moonlight-android/, create a file called ‘local.properties’. Add an ‘ndk.dir=’ property to the local.properties file and set it equal to your NDK directory.
-* Build the APK using Android Studio or gradle
+## Build
+
+1. Install [Android Studio](https://developer.android.com/studio)
+1. Install the Android NDK
+   1. Determine which version to use by finding `ndkVersion` in
+      [`app/build.gradle`](app/build.gradle)
+   1. In Android Studio, open _Tools»SDK Manager_
+      ![image](ndk.png)
+   1. Click the _SDK Tools_ tab
+   1. Enable the _Show Package Details_ checkbox in the bottom-right
+   1. Expand the _NDK (Side by side)_ category
+   1. Enable the checkbox for the version determined above
+   1. Click OK
```

Comment:
For me, it automatically installed the NDK version that was specified in the gradle file, but this is probably still helpful information.